### ISSUE TITLE
[QUESTION] Log only if action took place

### ISSUE THREAD

**Issue by h-e-l-l-o-w-o-r-l-d** [cobwebch/external_import#179]
Hi,

we run the scheduler every 5 minutes to check for new/updated records. This leads to a lot of logs where nothing happened. These logs are not interesting for us and in the log module it's a time-waster to spot logs where action took place.

Is it possible to only log if either insert, update, delete OR move is greater than 0?

**Comment by fsuter** [comment]
I can see the problem. This is currently not possible. It would need a new option.

BTW note that the number of updates operations is also not always correct, because I'm counting the number of records that are prepared for update, but the DataHandler will not save records that are identical and provides no reporting about this. So the number of records that is announced for updating may not reflect reality. This is complicated, because it means External Import would have to read the existing data to assess effective changes.

**Comment by h-e-l-l-o-w-o-r-l-d** [comment]
Yeah, I'm aware of that. Not a big problem though. Changed the issue-title from question to request. ;)
